### ISSUE TITLE
Lutris has a tendency to hang when install Windows GOG games

### ISSUE THREAD

**Issue by geajack** [lutris/lutris#4764]
Quite often, when I install a Windows game from GOG (using a local installer file that I downloaded in a browser), the installer window hangs in this state:

![image](https://user-images.githubusercontent.com/2124157/222918739-2bc3b7b0-e885-4526-b544-4eab97db79ce.png)

The GOG installer window itself is invisible at this point (which is probably a separate bug) - it appears in the Ubuntu dock (sidebar), but if I click on it I can't actually bring it up.

I'm pretty sure the installation is done, because in a terminal I can see that nothing in the wineprefix has been modified in twenty minutes.

This is likely at least partly an upstream Wine issue, but I'm opening this ticket to track it here anyway, because it's a very visible problem for the Lutris user experience - I experienced it multiple times in my first few days trying to use Lutris.

**Comment by danieljohnson2** [comment]
I've put a bunch of work into improving this window; perhaps you could install the recently released 0.5.13 beta and try that? Maybe it will work better. Let us know.

If it isn't fixed, please post the Lutris debug log after trying it. Run "lutris -d", install a game that hangs like this, and give us the terminal spam it puts out. This kind of thing, I expect a trace-back in there.

**Comment by geajack** [comment]
The problem is still occurring, but it's definitely a Wine issue. The real problem is essentially that once the GOG installer window disappears (say I minimize it, or I think even if I put something in front of it), it's just gone, and I can't convince my window manager to show it to me again. The issue is probably still worth tracking here, maybe there's some weird change that can be made to the way Lutris calls Wine.

**Comment by danieljohnson2** [comment]
I think I've seen this too, not long ago. I mean, those GOG installer windows are trouble and they never show in the activities overview. Sometimes the dash works, sometimes not.

But, for a while the GOG installer windows would not show *at all*, though it was running. That seemed to be an issue with a specific version of Wine-Proton as fermented by @GloriousEggroll, but whatever that was it seems to have got fixed. At least it is working for me now.

But this was not Lutris hanging; it wasn't GOG hanging either.  Everything was running, but with an invisible window it was just not useful.

A workaround I found then was to use an older, more boring Wine to install, then upgrade later. ``lutris-7.2-2-x86-64`` worked for me. Maybe you should try installing with that version?

To do that, set it as the default version in the Wine runner settings before installing. You'll need a fairly recent Lutris for that, but 0.5.13 should work.

**Comment by geajack** [comment]
> But this was not Lutris hanging; it wasn't GOG hanging either. Everything was running, but with an invisible window it was just not useful.

Yes, to be clear, the "hanging" is a symptom of the window being gone and, invisibly, waiting for you to hit the "Finish" button.

**Comment by strycore** [comment]
KDE Plasma isn't affected by this issue, I've only noticed it on Gnome.

**Comment by danieljohnson2** [comment]
I strongly suspect this is not a Lutris bug at all; maybe Wine, maybe Mutter, maybe some unholy tryst 'tween the two.

I've seen this, but even on GNOME it's not something I can reproduce now.

**Comment by hero2zer0** [comment]
I get this practically for every game I try to install - Darkwood, Underrail, Days Gone. It does eventually finish but it takes a long while...

**Comment by legluondunet** [comment]
can not reproduce
